sonar-orchestrator/src/main/java/com/sonar/orchestrator/locator/URLLocator.java

```diff
@@ -100,33 +96,10 @@ private static boolean isHttpRequest(URLLocation location) {
     return location.getURL().getProtocol().toLowerCase(Locale.ENGLISH).startsWith("http");
   }
 
-  private static Response sendHttpRequest(URLLocation location) throws IOException {
+  private static HttpResponse callHttpRequest(URLLocation location) throws IOException {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from method's body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)